x-pack/plugins/security_solution/server/fleet_integration/handlers/create_event_filters.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import uuid from 'uuid';
+import { i18n } from '@kbn/i18n';
+import { ENDPOINT_EVENT_FILTERS_LIST_ID } from '@kbn/securitysolution-list-constants';
+import type { Logger } from '@kbn/core/server';
+import type { PackagePolicy } from '@kbn/fleet-plugin/common';
+import type { ExceptionListClient } from '@kbn/lists-plugin/server';
+import type { EventFilters } from '../types';
+
+const PROCESS_INTERACTIVE_ECS_FIELD = 'process.entry_leader.interactive';
+
+/**
+ * Check if Event Filter list exists and Create Event Filters for the Elastic Defend integration.
+ */
+export const createEventFilters = async (
+  logger: Logger,
+  exceptionsClient: ExceptionListClient,
+  eventFilters: EventFilters,
+  packagePolicy: PackagePolicy
+) => {
+  const exceptionList = await exceptionsClient.getExceptionList({
+    id: undefined,
+    listId: ENDPOINT_EVENT_FILTERS_LIST_ID,
+    namespaceType: 'agnostic',
+  });
+
+  if (exceptionList == null) {
+    logger.error(
+      `Error: Exception List not found: ${ENDPOINT_EVENT_FILTERS_LIST_ID}. Event Filter not created.`
+    );
+  } else if (eventFilters.nonInteractiveSession) {
```

Comment:
This should probably be the first check this function makes and exit immediately  if its `false`